common/walker/src/report.rs

```diff
@@ -124,8 +128,83 @@ impl SplitScannerError for Result<Report, ScannerError> {
 }
 
 /// Handle the report
-pub async fn handle_report(report: Report) -> anyhow::Result<()> {
-    // FIXME: this is a very simplistic version of handling the error
-    log::info!("Import report: {report:#?}");
+pub async fn handle_report(report: Report, report_type: String) -> anyhow::Result<()> {
+    let is_report = env::var_os("IS_REPORT").unwrap_or_else(|| OsString::from(DEFAULT_IS_REPORT));
```

Comment:
Command line arguments like this (including env-vars) should be handled by `clap`.

Reply:
if we use  ``` clap,

Reply:
if we use `clap`, we need to do the same logic in bombastic and vexination. I want to control the changes in common report.

Reply:
Right now we can do `--help` and see what is supported. We also have a dedicated place for each command's arguments.

Spreading env-var processing throughout the application will make it terribly difficult to get an understanding of what is supported.

Reply:
ok